.github/workflows/Testear_versiones.yml

```diff
@@ -0,0 +1,29 @@
+name: Testear Versiones Python
+
+on:
+  push:
+    branches:
+      - main
+
+  pull_request:
+    branches:
+      - main
+      
+
+jobs:
+  test_python:
+    strategy:
+      matrix:
+        python_version: ["3.6","3.10"]
```

Comment:
Estás testeando exactamente las mismas versiones que en Circle CI. Es decir, est test es inútil.